src-ghc/Pact/ReplTools.hs

```diff
@@ -123,8 +123,8 @@ handleMultilineInput input prevLines lastResult =
            -- If so, continue accepting input
            -> haskelineLoop multilineInput lastResult
 
-       Failure e -> do
-         liftIO $ print e
```

Comment:
unnecessary change from `$` to parens here. What's the before/after for this PR?

Reply:
Thanks Robert! LGTM